src/mkdocs_coverage/plugin.py

```diff
@@ -53,31 +52,30 @@ def on_files(self, files: Files, config: MkDocsConfig, **kwargs: Any) -> Files:
         Returns:
             The modified files collection.
         """
-        page_name = self.config.page_name
-        page_path: str
-        if page_name is not None:
-            warnings.warn(
-                "The 'page_name' configuration option is deprecated and will be removed in a future release. "
-                "Use the 'page_path' configuration option instead.",
-                DeprecationWarning,
-                stacklevel=1,
-            )
-            page_path = page_name
-        else:
-            page_path = self.config.page_path
-        self.page_path = page_path
-        covindex = "covindex.html" if config.use_directory_urls else f"{page_path}/covindex.html"
+        self.page_path = self.config.page_path if self.config.page_name is None else self.config.page_name
```

Comment:
Isn't `self.config.page_name` moved to `self.config.page_path` by MkDocs anyway? Can't we therefore just do:


```suggestion
        self.page_path = self.config.page_path
```

Reply:
Thanks for the reminder. Nice catch!